spaces/S000063/properties/P000061.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000063
+property: P000061
+value: false
+---
+
+The seubset $A=\mathbb Q+\sqrt{2}$ is open and $F_\sigma$ in $X$, therefore a cozero set, since {S63|P7}.
+
+Suppose $B$ is a cozero set disjoint from $A$ such that $A\cup B$ is dense. Since $\mathbb Q\subset \overline{A}$, we have $B\subseteq\mathbb R\setminus\mathbb Q$.
+By the density of $B\cup A$ we know that $A\cup B=\mathbb R\setminus \mathbb Q$ and consequently, the irrationals are
+$F_\sigma$. Since neighbourhoods of rational numbers always contain the Euclidean ones, it would imply $\mathbb Q$ is a $G_\delta$ subset of {S25},
+contradicting the Baire category theorem.
```

Comment:
```suggestion
refs:
- zb: "0684.54001"
  name: General Topology (Engelking, 1989)
---

The subset $A=\mathbb Q+\sqrt{2}$ is open and an $F_\sigma$ set in $X$, therefore a cozero set, since {S63|P13} (see Corollary 1.5.13 in {{zb:0684.54001}}).

Suppose $B$ is a cozero set disjoint from $A$ such that $A\cup B$ is dense. Since $\mathbb Q\subset \overline{A}$, we have $B\subseteq\mathbb R\setminus\mathbb Q$.
By the density of $A\cup B$ we know that $A\cup B=\mathbb R\setminus \mathbb Q$ and consequently, the irrationals are
$F_\sigma$. Since neighbourhoods of rational numbers always contain Euclidean neighbourhoods, it would imply $\mathbb Q$ is a $G_\delta$ subset of {S25},
contradicting the Baire category theorem.
```